### PR TITLE
feat: updated endpoints for posts

### PR DESCRIPTION
**Pull request by m-atia**

### CODE REVIEW

File: wp/wp-content/themes/phila.gov-theme/js/dev/phila-gov.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = $(function () {
 
     var drilldownOptions = {
       autoHeight: false,
-      scrollTop: true,
+      scrollTop: false,
```
Comment (karissademi) [inline wp/wp-content/themes/phila.gov-theme/js/dev/phila-gov.js:17]: Are we 100% sure that we need to change this value and that it won't have negative effects on the mobile menu on other pages? Let's confirm before merge.

**Review by karissademi — APPROVED** [review]
There's just one JS change to check before merging this. Also, check merging Ritika's blog post branch into this branch **before merging this into staging**. There have been some changes to the advanced blog posts work that will likely conflict with this branch.

File: wp/wp-content/themes/phila.gov-theme/js/dev/phila-gov.js (hunk 1 above)
Comment (m-atia) [reply wp/wp-content/themes/phila.gov-theme/js/dev/phila-gov.js]: I undid this, I think it was a relic of debugging another bug.

**Comment by m-atia** [comment]
I merged the blog_posts branch into blog-endpoints, which should have support for the advanced blog post type.

**Pull request merged**